src/bootstrap.ts

```diff
@@ -0,0 +1,400 @@
+import Errors from './errors'
+import * as crypto from 'crypto'
+
+import {
+  type QueryParams,
+  type Bootstrap,
+  type BootstrapPage,
+  type Response,
+  type Channel
+} from './defs'
+
+export default class Bootstraps {
+  // Bootstraps API Client
+  /**
+   * @class Bootstrap
+   * Bootstrap is used to create, update, view and remove bootstrap configurations.
+   * @param {string} bootstraps_url - The url of the bootstraps service.
+   * @param {string} content_type - The content type of the request.
+   * @param {string} bootstrapsEndpoint - The endpoint of the bootstraps service which is
+   * configs.
+   * @returns {Bootstrap} - Returns a Bootstrap object.
+   *
+   */
+
+  private readonly bootstrapsUrl: URL
+  private readonly contentType: string
+  private readonly bootstrapsEndpoint: string
+  private readonly bootstrapError: Errors
+
+  public constructor (bootstrapsUrl: string) {
+    this.bootstrapsUrl = new URL(bootstrapsUrl)
+    this.contentType = 'application/json'
+    this.bootstrapsEndpoint = 'configs'
```

Comment:
```suggestion
    this.bootstrapsEndpoint = 'things/bootstrap'
    this.configsEndpoint = 'things/configs'
    this.whitelistEndpoint      = "things/state"
    this.bootstrapCertsEndpoint = "things/configs/certs"
    this.bootstrapConnEndpoint  = "things/configs/connections"
    this.secureEndpoint         = "secure"
```

then replace these in the urls in this file